### PR TITLE
Cards page

### PR DESCRIPTION
**Pull request by britt6612**
Add `Cards` page

### CODE REVIEW

File: src/screens/Cards.js [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+          badge:
+            'https://cdn-images-1.medium.com/fit/c/120/120/1*TD1P0HtIH9zF0UEH28zYtw.png',
+          label: 'Storybook',
+        },
+        {
+          url: 'https://storybook.grommet.io/?selectedKind=Layout-Card&full=0&stories=1&panelRight=0',
+          badge:
+            'https://cdn-images-1.medium.com/fit/c/120/120/1*TD1P0HtIH9zF0UEH28zYtw.png',
+          label: 'Storybook',
+        },
```
Comment (jcfilben) [inline src/screens/Cards.js:43]: Remove extra storybook link
```suggestion
```

File: src/screens/Cards.js [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+            <Example>{`{(datum) => (
+17        <Card key={datum.city} as="li">
+18          <CardBody>
+19            <Heading level={2} size="small" margin="none">
+20              {datum.city}
+21            </Heading>
+22          </CardBody>
+23          <CardFooter>{datum.state}</CardFooter>
+24        </Card>
+25      )}`}</Example>
```
Comment (jcfilben) [inline src/screens/Cards.js:99]:
```suggestion
            <Example>{`{(datum) => (
  <Card key={datum.city} as="li">
    <CardBody>
      <Heading level={2} size="small" margin="none">
        {datum.city}
      </Heading>
    </CardBody>
    <CardFooter>{datum.state}</CardFooter>
  </Card>
)}`}</Example>
```

File: src/screens/Cards.js [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+          <Description>How many items to render at a time.</Description>
+          <PropertyValue type="number">
+            <Example defaultValue>50</Example>
+          </PropertyValue>
+        </Property>
+      </Properties>
+      <ThemeDoc>
+        <Property name="cards.container">
+          <Description>Any Box properties.</Description>
+          <Example defaultValue> {`{}`} </Example>
```
Comment (jcfilben) [inline src/screens/Cards.js:201]:
```suggestion
          <PropertyValue type="object">
            <Example defaultValue> {`{}`} </Example>
          </PropertyValue>
```

**Review by jcfilben — APPROVED** [review]
Looks good!

**Review by ericsoderberghp — APPROVED** [review]

**Pull request merged**